web_src/js/utils/dom.js

```diff
@@ -65,10 +65,14 @@ export function queryElemSiblings(el, selector = '*', fn) {
 }
 
 // it works like jQuery.children: only the direct children are selected
-export function queryElemChildren(parent, selector = '*', fn) {
```

Comment:
Remove on line 63 as well.

Reply:
No. They are different.

`queryElemChildren` shouldn't have `selector = '*'`, because `el.children` already provides the children.

But, there is no `el.siblings`, so the call to `queryElemSiblings` could omit the "selector", but the call to "queryElemChildren" shouldn't omit the selector (otherwise, just use children)

Reply:
There are real calls: `hideElem(queryElemSiblings(target));`